change/@fluentui-react-tags-62b2fcad-e929-416b-88a0-1f6617266d80.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30973&commitId=255fa28442cec721a15b92124a11745ac6db6600&env=prod&clientType=FLUENTUI)


  <details><summary><b>InteractionTag Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand - Dark Mode.hover content.chromium.png|1| Changed|
|InteractionTag Converged.Brand.default.chromium.png|9| Changed|
|InteractionTag Converged.Brand - Dark Mode.pressed dismiss.chromium.png|14| Changed|
|InteractionTag Converged.Brand - Dark Mode.hover dismiss.chromium.png|14| Changed|
|InteractionTag Converged.Brand - Dark Mode.pressed content.chromium.png|1| Changed|
|InteractionTag Converged.Brand - Dark Mode.default.chromium.png|16| Changed|
|InteractionTag Converged.Brand.hover dismiss.chromium.png|9| Changed|
|InteractionTag Converged.Brand.pressed dismiss.chromium.png|9| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>